lib/transaction/Transactions.ts

```diff
@@ -1,4 +1,5 @@
 import BigNumber from 'bignumber.js';
+import { AdmissionControlStatus } from '../__generated__/admission_control_pb';
```

Comment:
Please can you remove this import from here. I don't think it is used and as a principle I want to limit usage of the `__generated__` classes to only the `client` module.

Reply:
Ok.